docs/requirement-sets/common/nested-app-auth-requirement-sets.md

```diff
@@ -28,12 +28,12 @@ NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identi
 | Application | Web        | Windows                                              | Mac        | iOS/iPad           | Android        |
 |-------------|------------|------------------------------------------------------|------------|--------------------|----------------|
 | Excel       | In preview | In preview                                           | In preview | In preview on iPad | Not applicable |
-| Outlook     | GA         | GA in Current Channel, Preview in all other channels | GA         | GA (iOS)           | GA             |
+| Outlook     | GA         | GA in Current Channel and Monthly Enterprise Channel, Preview in Semi-Annual Channels | GA         | GA (iOS)           | GA             |
```

Comment:
Consider updating the ms.date attribute of this article.

Reply:
done